src/MainWindow.cpp

```diff
@@ -56,14 +62,30 @@ bool MainWindow::onWindowStateEvent(GdkEventWindowState* event)
     return false;
 }
 
+bool MainWindow::onClose(GdkEventAny* event)
+{
+    if ( m_closetotray == true ) {
```

Comment:
When I close the window when tray is active, window is not being hidden, maybe a problem of i3wm with appindicators but needs to be checked.

Reply:
I couldn't find the correct solution for hiding the window, because the application is tied to this window and when it is hidden by function hide(), the application stops. I think that it is necessary to unbind the application from the window, then it will be possible to hide it correctly.
Now I use `set_skip_taskbar_hint(true)` to hide the icon in the taskbar and `iconify()` to minimize the window. This is the wrong solution, but it works in GNOME.

Reply:
Test it now please

Reply:
I am pretty busy this week. Will test it soon. Thanks.